starter-code/src/functions-and-arrays.js

```diff
@@ -34,6 +71,13 @@ const wordsArr = [
 ];
 
 // Unique Arrays
```

Comment:
Pour vérifier que le mot n'existe pas, ton indexOf === -1
Exemple de solution : 
function uniquifyArray(array) {
  var newArray = [];
  if (array.length === 0) {
    return undefined;
  }
  for (var i = 0; i < array.length; i++) {
    if (newArray.indexOf(array[i]) === -1) {
      newArray.push(array[i]);
    }
  }
  return newArray;
}